packages/flutter/lib/src/rendering/sliver_persistent_header.dart

```diff
@@ -178,7 +178,7 @@ abstract class RenderSliverPersistentHeader extends RenderSliver with RenderObje
     }());
     double stretchOffset = 0.0;
     if (stretchConfiguration != null && childMainAxisPosition(child) == 0.0)
-      stretchOffset += constraints.overlap.abs();
+      stretchOffset += constraints.overlap.abs().floorToDouble();
```

Comment:
usually, we fix this error by using https://api.flutter.dev/flutter/foundation/precisionErrorTolerance-constant.html
in the `if` condition check

Reply:
So I've tried this with the precisionErrorTolerance constant, and it looks like the issue still persists. I think we'll need a value higher than the current constant for this bug.

Reply:
Here is a clip: 

![ezgif com-video-to-gif (4)](https://user-images.githubusercontent.com/16964204/78702948-44b42f00-78be-11ea-9958-e4ab879157be.gif)

Reply:
I apply this fix locally, but i still see the same issue. Am i doing something wrong?
![sliverheader](https://user-images.githubusercontent.com/47866232/78705175-f143e000-78c1-11ea-9556-22ad9e9bc764.gif)

Reply:
Oh wow. That is bizarre. Let me try it again locally.

Reply:
Stranger still. It looks like the fix works on an iPhone 11 simulator, but not a Pixel 2 emulator. I'll continue to investigate.